### ISSUE TITLE
pact-jvm-provider_2.11 doesn't work with Maven 3.5.0

### ISSUE THREAD

**Issue by bodiam** [DiUS/pact-jvm#453]
When executing mvn pact-jvm-provider_2.11:verify (mvn pact:verify doesn't work?), I get the following error when using Maven 3.5.0. Maven 3.2.5 works.

```
Apache Maven 3.5.0 (ff8f5e7444045639af65f6095c62210b5713f426; 2017-04-04T05:39:06+10:00)
Maven home: C:\Dev\apache-maven-3.5.0\bin\..
Java version: 1.8.0_131, vendor: Oracle Corporation
Java home: C:\Program Files\Java\jdk1.8.0_131\jre
Default locale: en_AU, platform encoding: Cp1252
OS name: "windows 8.1", version: "6.3", arch: "amd64", family: "windows"
```

Error output:

```
ERROR] Failed to execute goal au.com.dius:pact-jvm-provider-maven_2.11:3.3.1:verify (default-cli) on project Valuations: Execution default-cli of goal au.com.dius:pact-jvm-provider-maven_2.11:3.3.1:verify failed: A required class was missing while executing au
.com.dius:pact-jvm-provider-maven_2.11:3.3.1:verify: org/fusesource/jansi/internal/CLibrary
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>au.com.dius:pact-jvm-provider-maven_2.11:3.3.1
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/C:/dev/workspace/localrepository/au/com/dius/pact-jvm-provider-maven_2.11/3.3.1/pact-jvm-provider-maven_2.11-3.3.1.jar
[ERROR] urls[1] = file:/C:/dev/workspace/localrepository/org/scala-lang/scala-library/2.11.8/scala-library-2.11.8.jar
[ERROR] urls[2] = file:/C:/dev/workspace/localrepository/org/scala-lang/scala-compiler/2.11.8/scala-compiler-2.11.8.jar
[ERROR] urls[3] = file:/C:/dev/workspace/localrepository/org/scala-lang/scala-reflect/2.11.8/scala-reflect-2.11.8.jar
[ERROR] urls[4] = file:/C:/dev/workspace/localrepository/org/scala-lang/modules/scala-xml_2.11/1.0.4/scala-xml_2.11-1.0.4.jar
[ERROR] urls[5] = file:/C:/dev/workspace/localrepository/org/scala-lang/modules/scala-parser-combinators_2.11/1.0.4/scala-parser-combinators_2.11-1.0.4.jar
[ERROR] urls[6] = file:/C:/dev/workspace/localrepository/com/typesafe/scala-logging/scala-logging_2.11/3.4.0/scala-logging_2.11-3.4.0.jar
[ERROR] urls[7] = file:/C:/dev/workspace/localrepository/au/com/dius/pact-jvm-provider_2.11/3.3.1/pact-jvm-provider_2.11-3.3.1.jar
[ERROR] urls[8] = file:/C:/dev/workspace/localrepository/au/com/dius/pact-jvm-model/3.3.1/pact-jvm-model-3.3.1.jar
[ERROR] urls[9] = file:/C:/dev/workspace/localrepository/org/json/json/20160212/json-20160212.jar
[ERROR] urls[10] = file:/C:/dev/workspace/localrepository/io/netty/netty-handler/4.0.40.Final/netty-handler-4.0.40.Final.jar
[ERROR] urls[11] = file:/C:/dev/workspace/localrepository/io/netty/netty-buffer/4.0.40.Final/netty-buffer-4.0.40.Final.jar
[ERROR] urls[12] = file:/C:/dev/workspace/localrepository/io/netty/netty-common/4.0.40.Final/netty-common-4.0.40.Final.jar
[ERROR] urls[13] = file:/C:/dev/workspace/localrepository/io/netty/netty-transport/4.0.40.Final/netty-transport-4.0.40.Final.jar
[ERROR] urls[14] = file:/C:/dev/workspace/localrepository/io/netty/netty-codec/4.0.40.Final/netty-codec-4.0.40.Final.jar
[ERROR] urls[15] = file:/C:/dev/workspace/localrepository/com/github/zafarkhaja/java-semver/0.9.0/java-semver-0.9.0.jar
[ERROR] urls[16] = file:/C:/dev/workspace/localrepository/au/com/dius/pact-jvm-matchers_2.11/3.3.1/pact-jvm-matchers_2.11-3.3.1.jar
[ERROR] urls[17] = file:/C:/dev/workspace/localrepository/org/apache/commons/commons-lang3/3.4/commons-lang3-3.4.jar
[ERROR] urls[18] = file:/C:/dev/workspace/localrepository/io/gatling/jsonpath_2.11/0.6.4/jsonpath_2.11-0.6.4.jar
[ERROR] urls[19] = file:/C:/dev/workspace/localrepository/org/scalatest/scalatest_2.11/2.2.6/scalatest_2.11-2.2.6.jar
[ERROR] urls[20] = file:/C:/dev/workspace/localrepository/commons-io/commons-io/2.5/commons-io-2.5.jar
[ERROR] urls[21] = file:/C:/dev/workspace/localrepository/org/codehaus/groovy/groovy-all/2.4.7/groovy-all-2.4.7.jar
[ERROR] urls[22] = file:/C:/dev/workspace/localrepository/org/codehaus/groovy/modules/http-builder/http-builder/0.7.1/http-builder-0.7.1.jar
[ERROR] urls[23] = file:/C:/dev/workspace/localrepository/net/sf/json-lib/json-lib/2.3/json-lib-2.3-jdk15.jar
[ERROR] urls[24] = file:/C:/dev/workspace/localrepository/commons-beanutils/commons-beanutils/1.8.0/commons-beanutils-1.8.0.jar
[ERROR] urls[25] = file:/C:/dev/workspace/localrepository/commons-collections/commons-collections/3.2.1/commons-collections-3.2.1.jar
[ERROR] urls[26] = file:/C:/dev/workspace/localrepository/commons-lang/commons-lang/2.4/commons-lang-2.4.jar
[ERROR] urls[27] = file:/C:/dev/workspace/localrepository/net/sf/ezmorph/ezmorph/1.0.6/ezmorph-1.0.6.jar
[ERROR] urls[28] = file:/C:/dev/workspace/localrepository/net/sourceforge/nekohtml/nekohtml/1.9.16/nekohtml-1.9.16.jar
[ERROR] urls[29] = file:/C:/dev/workspace/localrepository/xerces/xercesImpl/2.9.1/xercesImpl-2.9.1.jar
[ERROR] urls[30] = file:/C:/dev/workspace/localrepository/xml-apis/xml-apis/1.3.04/xml-apis-1.3.04.jar
[ERROR] urls[31] = file:/C:/dev/workspace/localrepository/xml-resolver/xml-resolver/1.2/xml-resolver-1.2.jar
[ERROR] urls[32] = file:/C:/dev/workspace/localrepository/org/apache/httpcomponents/httpclient/4.5.2/httpclient-4.5.2.jar
[ERROR] urls[33] = file:/C:/dev/workspace/localrepository/org/apache/httpcomponents/httpcore/4.4.4/httpcore-4.4.4.jar
[ERROR] urls[34] = file:/C:/dev/workspace/localrepository/commons-logging/commons-logging/1.2/commons-logging-1.2.jar
[ERROR] urls[35] = file:/C:/dev/workspace/localrepository/commons-codec/commons-codec/1.9/commons-codec-1.9.jar
[ERROR] urls[36] = file:/C:/dev/workspace/localrepository/com/googlecode/java-diff-utils/diffutils/1.3.0/diffutils-1.3.0.jar
[ERROR] urls[37] = file:/C:/dev/workspace/localrepository/org/reflections/reflections/0.9.10/reflections-0.9.10.jar
[ERROR] urls[38] = file:/C:/dev/workspace/localrepository/com/google/guava/guava/15.0/guava-15.0.jar
[ERROR] urls[39] = file:/C:/dev/workspace/localrepository/org/javassist/javassist/3.19.0-GA/javassist-3.19.0-GA.jar
[ERROR] urls[40] = file:/C:/dev/workspace/localrepository/com/google/code/findbugs/annotations/2.0.1/annotations-2.0.1.jar
[ERROR] urls[41] = file:/C:/dev/workspace/localrepository/net/databinder/unfiltered-netty-server_2.11/0.8.4/unfiltered-netty-server_2.11-0.8.4.jar
[ERROR] urls[42] = file:/C:/dev/workspace/localrepository/net/databinder/unfiltered-netty_2.11/0.8.4/unfiltered-netty_2.11-0.8.4.jar
[ERROR] urls[43] = file:/C:/dev/workspace/localrepository/net/databinder/unfiltered_2.11/0.8.4/unfiltered_2.11-0.8.4.jar
[ERROR] urls[44] = file:/C:/dev/workspace/localrepository/io/netty/netty-codec-http/4.0.24.Final/netty-codec-http-4.0.24.Final.jar
[ERROR] urls[45] = file:/C:/dev/workspace/localrepository/net/databinder/unfiltered-util_2.11/0.8.4/unfiltered-util_2.11-0.8.4.jar
[ERROR] urls[46] = file:/C:/dev/workspace/localrepository/net/databinder/dispatch/dispatch-core_2.11/0.11.3/dispatch-core_2.11-0.11.3.jar
[ERROR] urls[47] = file:/C:/dev/workspace/localrepository/com/ning/async-http-client/1.9.11/async-http-client-1.9.11.jar
[ERROR] urls[48] = file:/C:/dev/workspace/localrepository/io/netty/netty/3.10.0.Final/netty-3.10.0.Final.jar
[ERROR] urls[49] = file:/C:/dev/workspace/localrepository/org/codehaus/groovy/groovy-all/2.4.7/groovy-all-2.4.7-indy.jar
[ERROR] urls[50] = file:/C:/dev/workspace/localrepository/org/codehaus/plexus/plexus-utils/3.0.22/plexus-utils-3.0.22.jar
[ERROR] urls[51] = file:/C:/dev/workspace/localrepository/javax/enterprise/cdi-api/1.0/cdi-api-1.0.jar
[ERROR] urls[52] = file:/C:/dev/workspace/localrepository/javax/annotation/jsr250-api/1.0/jsr250-api-1.0.jar
[ERROR] urls[53] = file:/C:/dev/workspace/localrepository/org/eclipse/sisu/org.eclipse.sisu.inject/0.3.2/org.eclipse.sisu.inject-0.3.2.jar
[ERROR] urls[54] = file:/C:/dev/workspace/localrepository/org/codehaus/plexus/plexus-component-annotations/1.5.5/plexus-component-annotations-1.5.5.jar
[ERROR] urls[55] = file:/C:/dev/workspace/localrepository/org/apache/maven/plugin-tools/maven-plugin-annotations/3.4/maven-plugin-annotations-3.4.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
[ERROR]
[ERROR] -----------------------------------------------------
[ERROR] : org.fusesource.jansi.internal.CLibrary
[ERROR] -> [Help 1]
[ERROR]
```

**Comment by sanjimoh** [comment]
Hi, Found this issue while working with "pact-jvm-provider_2.12". Is there any workaround to make this working with maven 3.5.0?

**Comment by asangas** [comment]
Thanks for the great work guys. 

This issue is fixed with 3.5.11 right?

**Comment by uglyog** [comment]
Yes, it was released with version 3.5.0-rc3
